executable/build.go

```diff
@@ -53,11 +59,6 @@ func (b Build) Build(context libcnb.BuildContext) (libcnb.BuildResult, error) {
 
 	b.Logger.Title(context.Buildpack)
 
-	cr, err := libpak.NewConfigurationResolver(context.Buildpack, nil)
-	if err != nil {
-		return libcnb.BuildResult{}, fmt.Errorf("unable to create configuration resolver\n%w", err)
-	}
-
```

Comment:
```
	cr, err = libpak.NewConfigurationResolver(context.Buildpack, &b.Logger)
	if err != nil {
		return libcnb.BuildResult{}, fmt.Errorf("unable to recreate configuration resolver\n%w", err)
	}
```

Basically just add back in this returned code, with the slight difference that you pass in the logger.

Reply:
Fixed.